internal/api/front/perses/src/app/project/project-template/project-template.component.ts

```diff
@@ -25,7 +25,7 @@ import { ToastService } from '../../shared/service/toast.service';
 export class ProjectTemplateComponent implements OnInit {
   sidebarLinks = [
     {url: 'prometheusrules', icon: 'add_alert', label: 'Prometheus Rules'},
-    {url: 'dashboards', icon: 'dashboard', label: 'Dashboard'}
+    {url: 'dashboards', icon: 'insert_chart', label: 'Dashboards'}
```

Comment:
the icon `dashboard` was ok for me. Any reason to change it ?

Reply:
oh, actually insert_chart is the one I used on my side while you put dashboard on yours

IMHO insert_chart suits better / is less ambiguous, and we may use the dashboard one for sth else (I think it's used for alert templates in argos right ?)

@celian-garcia would be cool to have a 3rd PoV on this ! What do you prefer between
![image](https://user-images.githubusercontent.com/7058693/119858097-6e8e4200-bf14-11eb-97cf-a5e311e58045.png)
versus
![image](https://user-images.githubusercontent.com/7058693/119858013-5a4a4500-bf14-11eb-80be-5f2e07efedc4.png)
?

Reply:
discussed live yesterday